mne/io/fiff/tests/test_raw_fiff.py

```diff
@@ -411,22 +411,22 @@ def test_concatenate_raws_bads_order():
     # Bad channel mismatch raises
     raw2 = raw1.copy()
     raw2.info["bads"] = ["0", "2"]
-    with pytest.raises(ValueError):
+    with pytest.raises(ValueError, match="must match"):
         concatenate_raws([raw0, raw2])
 
     # Type mismatch raises
     epochs1 = make_fixed_length_epochs(raw1)
-    with pytest.raises(ValueError):
+    with pytest.raises(ValueError, match="type must match"):
         concatenate_raws([raw0, epochs1.load_data()])
 
     # Sample rate mismatch
     raw3 = _create_toy_data(sfreq=500)
-    with pytest.raises(ValueError):
+    with pytest.raises(ValueError, match="must match"):
```

Comment:
```suggestion
    with pytest.raises(ValueError, match="order must match"):
```

Reply:
This does not work @drammock. The error message is 
```Python
ValueError: raws[1].info['bads'] must match
```
since the sample rate does not match.

Reply:
Wait a second, the error message is wrong! I've corrected it in [2670811](https://github.com/mne-tools/mne-python/pull/11640/commits/2670811b9c56339fdfe4b37828a701e6cffc0758)

I can't write 
```Python
with pytest.raises(ValueError, match="raws[1].info['sfreq'] must match"):
    concatenate_raws([raw0, raw3])
```
because then the test fails.

Reply:
The error message is now correct though. I am not sure why matching `raws[1].info['sfreq']` does not work.

Reply:
Because what is matched is a regular expression (regex) the `[]` (character class delimiter in regex) need to be escaped not the quotes. You could also replace them with `.` which is regex for (almost) anything

Reply:
... and finally, a much faster way to iterate is too play with some online regex tool like https://regex101.com/ and see what works

Reply:
Thanks @larsoner, I changed the code! [09f444a](https://github.com/mne-tools/mne-python/pull/11640/commits/09f444a1894ec565dc6f957bdf12f67bef8a503e)